### ISSUE TITLE
Advanced and Universal Remotes from Simple Storage Network do not have a recipe.

### ISSUE THREAD

**Issue by Demlou6** [Darkosto/SkyFactory4Issues#33]
<img width="61" alt="screen shot 2018-10-16 at 7 04 23 am" src="https://user-images.githubusercontent.com/39318317/47015116-c126e100-d111-11e8-8a33-eaef9f790f1b.png">

**Comment by Darkosto** [comment]
Yeah, those should be turned off and not even show up in your game.

**Comment by Darkosto** [comment]
Did yours actually show up in JEI or does only the normal "Storage Remote" appear?
